src/main/resources/messages/RsBundle.properties

```diff
@@ -59,3 +59,8 @@ refactoring.change.signature.error.cfg.disabled.parameters=Cannot change signatu
 refactoring.change.signature.name.conflict=The name {0} conflicts with an existing item in {1}
 refactoring.change.signature.visibility.conflict=The function will not be visible from {0} after the refactoring
 refactoring.change.signature.refactor.super.function=Method {0} implements base method of trait {1}.\nDo you want to refactor the base method?
+
+intention.Rust.ToggleFeatureIntention.enable=Enable
+intention.Rust.ToggleFeatureIntention.disable=Disable
+intention.Rust.ToggleFeatureIntention.text={0} feature `{1}`
```

Comment:
Let's avoid such sentence construction because it complicates translation. Let's use a simple "Enable feature `{1}`" and "Disable feature `{1}`"